### PR TITLE
Markdown parser lib change, snarkdown instead of marked.

### PR DESCRIPTION
**Pull request by plesiecki**
https://github.com/allegro/turnilo/issues/845

https://bundlephobia.com/package/marked@4.0.12 vs https://bundlephobia.com/package/snarkdown@2.0.0

### CODE REVIEW

File: src/client/views/home-view/data-cube-card/data-cube-card.tsx [hunk 1]
```diff
@@ -48,8 +48,11 @@ export class DataCubeCard extends React.Component<DataCubeCardProps, DataCubeCar
 
   renderDescription() {
     const { description, extendedDescription } = this.props;
+    const MarkdownNode = React.lazy(() => import(/* webpackChunkName: "markdown-node" */ "../../../components/markdown-node/markdown-node"));
     if (!extendedDescription) {
-      return <MarkdownNode markdown={description || STRINGS.noDescription} />;
+      return <React.Suspense fallback={Loader}>
```
Comment (adrianmroz-allegro) [inline src/client/views/home-view/data-cube-card/data-cube-card.tsx:53]: Oh my!

Comment (plesiecki) [reply src/client/views/home-view/data-cube-card/data-cube-card.tsx:53]: [It works](https://turnilo-marked-gmbbyye42a-ew.a.run.app/) and how!

File: src/client/drag-and-drop-polyfill.ts [hunk 2]
```diff
@@ -23,9 +23,9 @@ const needsPatch = !(dragDiv || evts) || /iPad|iPhone|iPod|Android/.test(navigat
 if (needsPatch) {
   Promise.all([
     // @ts-ignore
-    import("../../lib/polyfill/drag-drop-polyfill.min.js"),
+    import(/* webpackChunkName: "dnd-js" */ "../../lib/polyfill/drag-drop-polyfill.min.js"),
     // @ts-ignore
-    import("../../lib/polyfill/drag-drop-polyfill.css")
+    import(/* webpackChunkName: "dnd-css" */ "../../lib/polyfill/drag-drop-polyfill.css")
```
Comment (adrianmroz-allegro) [inline src/client/drag-and-drop-polyfill.ts:28]: I assume these comments are meaningful for webpack? Will something horribly broke if someone removes them by accident?

**Review by adrianmroz-allegro — CHANGES_REQUESTED** [review]
Move button to Suspense boundary.

File: src/client/drag-and-drop-polyfill.ts (hunk 2 above)
Comment (plesiecki) [reply src/client/drag-and-drop-polyfill.ts:28]: It sets name for new chunk, otherwise webpack chunk index number as its name.

**Comment by plesiecki** [comment]
> Move button to Suspense boundary.

Done 👍

File: src/client/views/home-view/data-cube-card/data-cube-card.tsx [hunk 3]
```diff
@@ -58,7 +61,9 @@ export class DataCubeCard extends React.Component<DataCubeCardProps, DataCubeCar
     const action = showingMore ? this.showLess : this.showMore;
 
     return <React.Fragment>
-      <MarkdownNode markdown={content} />
+      <React.Suspense fallback={Loader}>
+        <MarkdownNode markdown={content} />
+      </React.Suspense>
       <div className="show-more-action" onClick={action}>{actionLabel}</div>
```
[the remark this answers is not in the document]
Comment (plesiecki) [reply src/client/views/home-view/data-cube-card/data-cube-card.tsx]: I like how you call this div a button 😅

[the remark this answers is not in the document]
Comment (adrianmroz-allegro) [reply src/client/views/home-view/data-cube-card/data-cube-card.tsx]: Does it "click"? So it is a button, duh!

**Review by adrianmroz-allegro — APPROVED** [review]
Conditional approve. Code is good, let's discuss with team if UX is acceptable.

File: test/setup/mocha.js [hunk 4]
```diff
@@ -13,25 +13,27 @@
...
-global.HTMLElement = global.window.HTMLElement;
-// setup for React
-global.navigator = {userAgent: "testing"};
+require("@babel/register")({
+    cache: true,
+    configFile: false,
+    extensions: [".ts", ".tsx", ".js", ".jsx"],
+    ignore: [
+        function(filePath) {
+            return filePath.includes("node_modules") && !filePath.includes("snarkdown")
```
[the remark this answers is not in the document]
Comment (plesiecki) [reply test/setup/mocha.js:23]: By default `ignore` is like `[/node_modules/]`. In the meantime, ESM module was imported under `snarkdown` and I had to treat it with a transpilation. But I re-checked this setting and it seems it's now all good without it 🤔

**Pull request merged**